c_glib/test/test-dictionary-encode.rb

```diff
@@ -23,19 +23,40 @@ def test_int32
     array = build_int32_array([1, 3, 1, -1, -3, -1])
     assert_equal(<<-STRING.chomp, array.dictionary_encode.to_s)
 
--- is_valid: all not null
--- dictionary: [1, 3, -1, -3]
--- indices: [0, 1, 0, 2, 3, 2]
+-- dictionary:
```

Comment:
Is `is_valid` intentionally missing here?

Reply:
It seems that new pretty print implementation doesn't show `is_valid` information.
Can you open a JIRA ticket to discuss what should we do for it?

Reply:
`is_valid` is a duplicate information. We also show nulls in the indices now, before the refactor we have shown undefined values instead there.

Reply:
I see. It makes sense.